activerecord/lib/active_record/query_logs.rb

```diff
@@ -130,7 +143,7 @@ def tag_content
             else
               handler
             end
-            "#{key}:#{val}" unless val.nil?
```

Comment:
Note how the legacy implementation coerces `val` into a string. I'm preserving this behavior in `QuotingFormatter.format_value` (see related comment)